internal/audit/report.go

```diff
@@ -40,5 +43,5 @@ type Report struct {
 	CgroupsPath    string
 	Status         string
 	Capabilities   *LinuxCapabilities
-	Devices        []*LinuxDevice
+	Devices        []LinuxDevice
```

Comment:
why not use `specsgo.LinuxDevice` instead of carrying a struct here?

Reply:
We used to point to specsgo.LinuxDevices, but that caused all devices to be copies of the first. Also, we need a local type, in order to define marshalling methods for Zap

Reply:
Sure, thanks for the feedback, appreciate it